digitalstrategie/assets/scss/components/_ds-block.scss

```diff
@@ -33,6 +33,18 @@
         .background-image + .content-container *:not(.button) {
             color: $black;
         }
+
+        @media print {
+            background-color: none !important;
```

Comment:
I know, nothing to do with this PR, but should we also set the text colour to black to make sure that the green blocks are also printed fine?